platformLibs/build.gradle

```diff
@@ -16,7 +16,7 @@ buildscript {
     }
 
     dependencies {
-        classpath "org.jetbrains.kotlin:kotlin-native-gradle-plugin"
```

Comment:
What about samples?

Reply:
They use the version specified in `samples/gradle.properties`. It is`+` for building samples in a work tree and is replaced with correct plugin version when a bundle is built.